src/api/get.js

```diff
@@ -0,0 +1,14 @@
+import Wreck from '@hapi/wreck'
+import { config } from '../config/config.js'
+import { createLogger } from '../common/helpers/logging/logger.js'
+
+const logger = createLogger()
+
+export async function get (url) {
+  try {
+    return await Wreck.get(`${config.get('backend.endpoint')}${config.get('backend.path')}${url}`)
+  } catch (error) {
+    logger.error(`Encountered error while calling the backend with url: ${config.backendEndpoint}${url}}`, error)
+    return null
```

Comment:
I would maybe just throw the error here.

Otherwise we're dealing with null being passed everywhere and we end up with a load more if statements and error handling.

If the API call has failed then we should just return an error to the user that the service isn't working and they should try again later.

The original pattern of checking if the API called in various models and functions feels wrong to me.